packages/block-library/src/home-link/edit.js

```diff
@@ -64,7 +50,7 @@ export default function HomeEdit( { attributes, setAttributes, context } ) {
 					<RichText
 						identifier="label"
 						className="wp-block-home-link__label"
-						value={ label }
+						value={ label || __( 'Home' ) }
```

Comment:
TBH I'm not sure we should change the editor-side behavior. I have a hunch that folks chose `useEffect` over a `||` fallback for a reason, which is the slightly funky UX we'd get if the user removes the label (e.g. by pressing backspace until nothing is left):

![home-link](https://github.com/WordPress/gutenberg/assets/96308/e7403a59-c5b6-44a3-823c-fa8f429f6ee5)

It's confusing enough that I'd rather keep the original code, even if that means that we'd allow an edge case where the user decides to remove the label, but the frontend falls back to 'Home'.

Reply:
Hmm good catch, this must have slipped by me as I would have done "CMD + A" and started typing a new label. I think what you have suggested is a good solution, leave the editor side as-is and just make sure _something_ renders ("Home") on the server if `label` is empty. I'll update this today.